src/index.css

```diff
@@ -0,0 +1,30 @@
+body {
+  background-color: rgb(250,250,250);
+  margin: 30px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+h2, h5{
+  margin: 20px 0 2px 0
+}
+
+p{
+  font-size: 12px;
+  margin: 5px 0  5px 20px
+}
+
+.country {
+  max-width: 50%;
+  padding: 5px;
+  margin: 0 0 20px 0
```

Comment:
can this just be `margin-bottom: 20px` here?

Reply:
Yes! I forgot change it, because in the begining I code other margin and I foget to change it! haha